pkg/activator/net/revision_backends.go

```diff
@@ -166,11 +166,6 @@ func (rw *revisionWatcher) probeClusterIP(dest string) (bool, error) {
 // not ready addresses) and return the ones that are successfully probed,
 // whether the update was a no-op, or an error.
 func (rw *revisionWatcher) probePodIPs(dests sets.String) (sets.String, bool, error) {
-	// Short circuit case where dests == healthyPods
```

Comment:
This was an important performance fix that we did. Why do we need to remove it?

Reply:
After this PR, the dests could also contain not ready address. In this case, should we probe them again?

Reply:
perhaps we need to merge them only within this function?

Reply:
I see. That makes more sense. I'll make the change.